appmonitoring/ts/src/server.ts

```diff
@@ -1,89 +1,107 @@
-﻿import * as fs from "fs";
-import * as https from "https";
+﻿import * as https from "https";
 import { ContentProcessor } from "./ContentProcessor.js";
 import { logger, Metrics } from "./LoggerWrapper.js";
 import { AppMonitoringConfigCR, IRootObject } from "./RequestDefinition.js";
 import { K8sWatcher } from "./K8sWatcher.js";
 import { AppMonitoringConfigCRsCollection } from "./AppMonitoringConfigCRsCollection.js"
+import fs from "fs";
+import { CertificateManager } from "./CertificateGenerator.js";
 
-const crs: AppMonitoringConfigCRsCollection = new AppMonitoringConfigCRsCollection();
 
-// don't await, this runs an infinite loop
-K8sWatcher.StartWatchingCRs((cr: AppMonitoringConfigCR, isRemoved: boolean) => {
-    if (isRemoved) {
-        crs.Remove(cr);
-    } else {
-        crs.Upsert(cr);
-    }
+const containerMode = process.env.CONTAINER_MODE;
+
+if ("secrets-manager".localeCompare(containerMode) === 0) {
+
+    console.log("Running in certificate manager mode...");
+    await CertificateManager.CreateWebhookAndCertificates();
```

Comment:
Need to account for a case when previous cleanup somehow didn't happen and objects that we are trying to create on the cluster somehow exist already. Handle that case gracefully.